Intersect (Core)/GameObjects/LightBase.cs

```diff
@@ -79,7 +83,9 @@ public override int GetHashCode()
         public override bool Equals(object obj)
         {
             if (obj is LightBase light)
```

Comment:
if (...) { ... }, no {} is bad -- also the return statement really doesn't need to be multiple lines, idk why your formatter split it like that